src/QMCWaveFunctions/EinsplineSetBuilder_createSPOs.cpp

```diff
@@ -146,6 +147,7 @@ SPOSet* EinsplineSetBuilder::createSPOSetFromXML(xmlNodePtr cur)
     a.add(truncate, "truncate");
     a.add(use_einspline_set_extended, "use_old_spline");
     a.add(myName, "tag");
+    a.add(skipChecks, "skipChecks");
```

Comment:
We would like to have QMCAPCK input entries all lower case. In this case, underscore helps to separate works. So please use "skip_checks".

Reply:
done